for/wordpress.md

```diff
@@ -0,0 +1,193 @@
+By default WPBrowser creates the `wpunit` suite to keep these tests separate from "real" unit tests stored in the `unit` suite.  
+The `wpunit` suite includes the `WPLoader` module: this takes care of loading, installing and configuring a fresh WordPress installation before each test method runs.  
+To handle the heavy lifting the module requires some information about the local WordPress installation: in the `codeception.yml` file configure it to match your local setup; with reference to the example above the module configuration might be:
+
+```yaml
+modules:
+    enabled:
+        WPLoader:
+            wpRootFolder: /var/www/wordpress
+            dbName: wordpress-tests
+            dbHost: localhost
+            dbUser: root
+            dbPassword: root
+            wpDebug: true
+            tablePrefix: wptests_
+            domain: wordpress.dev
+            plugins:
+                - acme-plugin/plugin.php
+            activatePlugins:
+                - acme-plugin/plugin.php
+```
+
+The module is wrapping and augmenting the [WordPress PHPUnit automated testing suite](https://make.wordpress.org/core/handbook/testing/automated-testing/phpunit/) and to generate a test case that uses Codeception and the methods provided by the Core testing suite you can use the `wpcept` command again:
+
+```bash
+wpcept generate:wpunit wpunit "Acme\SomeClass"
+```
+
+The generated test case extends the `Codeception\TestCase\WPTestCase` class and it exposes all the mehtods defined by `Codeception\TestCase\Test` test case and the Core suite `WP_UnitTestCase`.  
+Additional test method generation possibilities are available to cover the primitive test cases offered in the Core testing suite: `wpajax`, `wprest`, `wpcanonical`, `wpxmlrpc`.  
+Any database interaction is wrapped in a transaction to guarantee isolation between tests.
+
+<div class="alert alert-warning">
+  <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+  Continue to <a href="http://codeception.com/docs/05-UnitTests">Unit Testing Guide &raquo;</a>.
+</div>
+
+## WordPress Functional Tests
+Functional tests are meant to test complex interactions between a WordPress theme or plugin components, requests handling or a filter triggered chain of events.  
+A WordPress unit test might test the behaviour of a post insertion handling class in isolation where a functional test might verify the behaviour of said class when part of a chain of classes handling a post insertion request.  
+Similarly to WordPress Unit Tests the `functional` suite should include the `WPLoader` module to have WordPress running in the same scope as the test methods and the configuration is the same.
+Since the difference is subtle in infrastructure terms the command to generate a functional test is the same used to generate a WordPress unit test:
+
+```bash
+wpcept generate:wpunit functional "PostInsertionHandling"
+```
```

Comment:
sorry I didn't understand actual difference between functional and wpunit 
If there is no obvious distinction between them maybe they can be used in one suite?